lib/main.dart

```diff
@@ -22,7 +24,7 @@ class _MyAppState extends State<MyApp> {
 
   Widget build(BuildContext context) {
     return MaterialApp(
-        home: LoginPage()
+        home: SupplRegisterPage()
```

Comment:
Please do not use abbreviations and revert, will be handled when navigation is implemented.